src/main.rs

```diff
@@ -0,0 +1,187 @@
+extern crate rustc_serialize;
+extern crate docopt;
+#[macro_use]
+extern crate lazy_static;
+extern crate regex;
+extern crate tabwriter;
+
+mod benchmark;
+mod utils;
+
+use docopt::Docopt;
+use regex::Regex;
+use tabwriter::TabWriter;
+
+use benchmark::{Benchmark, parse_benchmarks};
+use utils::find_overlap;
+
+use std::io;
+use std::io::prelude::*;
+use std::fs::File;
+
+const USAGE: &'static str = r#"
+Compare Rust micro-benchmarks by saving the output of the benchmark to file
+and providing it into this command.
+The first version takes two file and compares the common bench-tests.
+The second version takes two names of implementations and one or more files,
```

Comment:
For whatever reason, I can't parse this description. I think I don't know what "name of implementation" means.